pkg/restore/restore.go

```diff
@@ -815,38 +815,41 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 			labels := obj.GetLabels()
 			addRestoreLabels(fromCluster, labels[api.RestoreNameLabel], labels[api.BackupNameLabel])
 
-			if !equality.Semantic.DeepEqual(fromCluster, obj) {
-				switch groupResource {
-				case kuberesource.ServiceAccounts:
-					desired, err := mergeServiceAccounts(fromCluster, obj)
-					if err != nil {
-						ctx.log.Infof("error merging secrets for ServiceAccount %s: %v", kube.NamespaceAndName(obj), err)
-						addToResult(&warnings, namespace, err)
-						continue
-					}
+			if equality.Semantic.DeepEqual(fromCluster, obj) {
+				ctx.log.Infof("Skipping restore of %v because it already exists in the cluster and is unchanged from the backed up version", name)
+				continue
```

Comment:
Instead of adding this `continue` statement, I suggest removing it and adding an `else` at the end of this statement. It's a small change that arguably makes it easier to read.